src/main/java/Duke.java

```diff
@@ -6,5 +11,87 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greetLine();
+        addList();
+        exitLine();
+    }
+    public static void greetLine(){
+        System.out.println("How may I be of service?");
+    }
+    public static void echo()
+    {
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        while(!line.equals("bye")) {
+            System.out.println(line);
+            line = in.nextLine();
+        }
+    }
+    public static void exitLine(){
+        System.out.println("Glad I could be of help!");
+    }
+
+    public static void addList()
+    {
+        Task[] list = new Task[100];
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        int numOfItems = 0;
+        while(!line.equals("bye")) { // condition to shut down program
+            if(line.equals("list")) { // users wants to know all text so far
+                printCurrentList(list, numOfItems);
+            }
+            else if(line.startsWith("mark")){
+                mark(line, list);
+                printCurrentList(list, numOfItems);
+            }
+            else if(line.startsWith("unmark")){
+                unmark(line,list);
+                printCurrentList(list, numOfItems);
+            }
+            else { // new tasks keyed in by user
+                Task newTask = new Task(line);
+                if(line.startsWith("todo")){
+                    newTask = new ToDos(line);
+                }
+                else if(line.startsWith("deadline")){
```

Comment:
change the form of the if/else statements to match the coding standard:

if (condition) {
    statements;
} else if (condition) {
    statements;
} else {
    statements;
}

Reply:
Yeah, you can refer to this link [https://se-education.org/guides/conventions/java/basic.html] for **if-else** statement coding standards.

Reply:
Thanks guys!